contracts/modules/token/TokenController.sol

```diff
@@ -19,18 +19,56 @@ import "@openzeppelin/contracts/math/SafeMath.sol";
 import "../../abstract/Iupgradable.sol";
 import "../../interfaces/IPooledStaking.sol";
 import "./NXMToken.sol";
-import "./external/IERC1132.sol";
 
-contract TokenController is IERC1132, Iupgradable {
+contract TokenController is Iupgradable {
```

Comment:
Make sure we didn't corrupt the storage by swapping the IERC1132 and Iupgradable state variables